### PR TITLE
Introduce get_data_from_new_media_translation_request()

### PR DESCRIPTION
**Pull request by Chouby**
Follows up #1532 
This PR introduces the new method `PLL_Admin_Links::get_data_from_new_media_translation_request()` similar to `PLL_Admin_Links::get_data_from_new_post_translation_request()` but for media.

### CODE REVIEW

File: admin/admin-links.php [hunk 1]
```diff
@@ -272,4 +276,47 @@ public function get_data_from_new_post_translation_request( string $post_type ):
...
+		$lang = $this->model->get_language( $lang_slug );
+
+		if ( empty( $post ) || empty( $lang ) ) {
+			return array();
+		}
+
+		return array(
+			'from_post' => $post,
+			'new_lang'  => $lang,
+		);
```
Comment (Screenfeed) [inline admin/admin-links.php:320]: Maybe we could share this part with `get_data_from_new_post_translation_request()`.

Ex:
```php
	private function get_translation_request_data_objects( int $post_id, string $lang_slug ): array {
		if ( $post_id <= 0 || empty( $lang_slug ) ) {
			return array();
		}

		$post = get_post( $post_id );
		$lang = $this->model->get_language( $lang_slug );

		if ( empty( $post ) || empty( $lang ) ) {
			return array();
		}

		return array(
			'from_post' => $post,
			'new_lang'  => $lang,
		);
	}
```

Comment (Hug0-Drelon) [reply admin/admin-links.php:320]: The returned key should be `from_media` instead of `from_post`, am I right?

**Review by Hug0-Drelon — CHANGES_REQUESTED** [review]

File: admin/admin-links.php (hunk 1 above)
Comment (Screenfeed) [reply admin/admin-links.php:320]: STOP RUINING EVERYTHING! 😭

Comment (Screenfeed) [reply admin/admin-links.php:320]: On second thought, we could keep `from_post` for both, it would require only few adjustments for the media 🤔

Comment (Hug0-Drelon) [reply admin/admin-links.php:320]: After working on https://github.com/polylang/polylang-pro/pull/2274 which depends on this very PR, I think we should keep it that way.
After all a media is a post, and it allows to reuse completely what's done in `PLL_ACF_Post`.

Comment (Chouby) [reply admin/admin-links.php:320]: I intentionally kept `from_post` for media too. I'm often annoyed with thinks like post `ID` but term `term_id` or other similar things in Polylang related to early decisions.

**Review by Hug0-Drelon — APPROVED** [review]

**Review by Hug0-Drelon — APPROVED** [review]

**Pull request merged**